backend/src/main/java/com/capstone/moneytree/utils/MoneyTreeError.java

```diff
@@ -0,0 +1,34 @@
+package com.capstone.moneytree.utils;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class MoneyTreeError {
+
+    private MoneyTreeErrorCode errorCode;
+    private String message;
```

Comment:
Here we have two messages. MoneyTreeErrorCode is a message itself. I think we should remove the message parameter.

Reply:
Also, I just thought about this and we're still missing a way to generate a status, message and body for the response all at the same time.. Remember that in the end, we'll return a ResponseEntity (with status needed and body) to the frontend as a final object when creating a user is completed for example.
So, I think we need a better way to handle this. What I am thinking right now is having a MoneyTreeBody class that will have:
- Error Code Message
- Body of any Object

Because we'll always be returning this:
`
return ResponseEntity
         .status(status)
         .body(moneyTreeBody)
`

And that body will have to have an object of any kind and a message for the frontend to know whats going on.